content/javascript/concepts/dates/terms/toLocaleDateString/toLocaleDateString.md

```diff
@@ -87,3 +87,23 @@ The output will be:
 ```shell
 Monday April 11, 2022
 ```
+
+## Codebyte Example
+
+In the following example the variables `region` and `options` can be modified, to print the Date in a custom format:
+
+```codebyte/javascript
+
```

Comment:
As noted above, please remove the additional empty lines at the top and bottom of the code block